app/src/main/kotlin/com/tailoredapps/androidapptemplate/navigation/Screen.kt

```diff
@@ -0,0 +1,8 @@
+package com.tailoredapps.androidapptemplate.navigation
+
+import androidx.annotation.StringRes
+import com.tailoredapps.androidapptemplate.R
+
+sealed class Screen(@StringRes val titleRes: Int, val route: String) {
+    object Overview : Screen(R.string.screen_overview, "overview")
```

Comment:
maybe add some example of multiple screen with navigation here

Reply:
including navigation parameters

Reply:
This is scope of the sample app, which will be refactored when this refactoring is finished

Reply:
👌